### DISCUSSION TITLE
about dwg to bim model by iTwin Snapshot

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#6849] [Q&A]
now , i test three dwg file to bim by  iTwin Snapshot
the three dwg files are  **same** and to bim file.
but just the main file show right  in bim viewer , and other file  show  error code in viewer!
  
![111111](https://github.com/iTwin/itwinjs-core/assets/18022720/41c43df2-b03a-4106-96d8-a703ef15d560)

![22222](https://github.com/iTwin/itwinjs-core/assets/18022720/a1b8c93e-ba06-41d6-bea9-f8d988716b1f)

**Comment by pmconne** [comment]
> but just the main file show right in bim viewer , and other file show error code in viewer!

I'm going to infer that the first screenshot is the one that shows "right" and that "error code" refers to the boxes where you expected to see text. Please clarify if I misunderstand.

You appear to be comparing two different views.
First screenshot:
![image](https://github.com/iTwin/itwinjs-core/assets/22944042/10c19827-6e85-4777-a3d2-52d5b0e8e835)
Second screenshot:
![image](https://github.com/iTwin/itwinjs-core/assets/22944042/68b0da01-5ba5-4e3d-abcf-066cb58340f3)

Please compare the exact same views in the two .bim files and see if there are any differences.

Those boxes generally indicate a font could not be found, or the font does not supply a particular glyph.

**Reply by paprophet** [reply]
the two different views from one bim file (three same dwg files to the bim file by iTwin Snapshot )
the problem is why other views has  the font does not supply a particular glyph 
note : the three  dwg files are same !

**Reply by pmconne** [reply]
Why would you import the same exact DWG file into 1 iModel 3 times?
Are you going to supply the DWG files? We don't have nearly enough information to diagnose the problem.
